app/services/user_access_key.rb

```diff
@@ -20,7 +20,9 @@ class UserAccessKey
 
   COST = Rails.env.test? ? SCRYPT_COST_MAX_TIME_0_DOT_01 : SCRYPT_COST_MAX_TIME_0_DOT_5
 
-  def initialize(password, salt)
+  def initialize(password, salt, cost = COST)
```

Comment:
how about I do that, and move the cost config to `application.yml`, in a follow-up PR? Trying to keep this small.